<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="mapTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Индекс:</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ключ:</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>581</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Множество:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="setLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="setResLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mapResLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Карта:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>851</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>                                  Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setInsertButton">
         <property name="text">
          <string>Вставить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setRemoveButton">
         <property name="text">
          <string>Удалить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setContainsButton">
         <property name="text">
          <string>Проверить, существует ли элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setClearButton">
         <property name="text">
          <string>Очистить множество</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>                                Map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapInsertButton">
         <property name="text">
          <string>Вставить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapRemoveButton">
         <property name="text">
          <string>Удалить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapContainsButton">
         <property name="text">
          <string>Проверить, существует ли элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapClearButton">
         <property name="text">
          <string>Очистить карту</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Индекс:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="keySpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Значение для карты:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="valSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Значение для множества:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="setValSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
